src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Avx2.PlatformNotSupported.cs

```diff
@@ -11,9 +11,9 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel AVX2 hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Avx2
+    public abstract class Avx2 : Avx
```

Comment:
14.7.1 Detection of AVX2

Hardware support for AVX2 is indicated by CPUID.(EAX=07H, ECX=0H):EBX.AVX2[bit 5]=1.

Application Software must identify that hardware supports AVX, after that it must also detect support for AVX2 by
checking CPUID.(EAX=07H, ECX=0H):EBX.AVX2[bit 5].